include/boost/url/encode.hpp

```diff
@@ -125,8 +127,8 @@ encoded_size(
     this parameter is omitted, the default
     options will be used.
 
-    @param allowed The set of characters
-    allowed to appear unescaped.
+    @param unescaped The set of characters
```

Comment:
If you are going to rename it then we have to rename it everywhere. `encoded_size_impl` above still says "allowed."